populus/compilation/backends/solc_standard_json.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import pprint
+
+from eth_utils import (
+    add_0x_prefix,
+    to_dict,
+)
+
+from solc import (
+    compile_standard,
+)
+from solc.exceptions import (
+    ContractsNotFound,
+)
+
+from populus.utils.contract_key_mapping import ContractKeyMapping
+
+from .base import (
+    BaseCompilerBackend,
+    _load_json_if_string,
+    _normalize_contract_metadata,
+    add_dependency_info,
+)
+
+
+@to_dict
+def _normalize_standard_json_contract_data(contract_data):
+    if 'metadata' in contract_data:
+        yield 'metadata', _normalize_contract_metadata(contract_data['metadata'])
+    if 'evm' in contract_data:
+        evm_data = contract_data['evm']
+        if 'bytecode' in evm_data:
+            yield 'bytecode', add_0x_prefix(evm_data['bytecode'].get('object', ''))
+            if 'linkReferences' in evm_data['bytecode']:
+                yield 'linkrefs', evm_data['bytecode']['linkReferences']
```

Comment:
I settled on a compromise for the compiled contract data format, but the link references internal structure is left the same as the standard-json output, as I understand the intention is to move towards using that.